scripts/schemas/sites.js

```diff
@@ -0,0 +1,39 @@
+const uriOrEmptyString = {
```

Comment:
It's only used ever here.

Reply:
We use it on L30 below. But since we only use it in the one place now, we can possibly just inline it instead of a separate variable.

Reply:
Yepp. One step at a time :)